upgrading/automated_upgrades.adoc

```diff
@@ -154,63 +228,93 @@ To prepare for an automated upgrade:
 # subscription-manager refresh
 ----
 
-. If you are upgrading from {product-title} 3.7 to 3.9:
+. For any upgrade path, always ensure that you have the latest version of the
+*openshift-ansible* package on each RHEL 7 system:
```

Comment:
`on your the host where you'll run the upgrade playbooks` ? It doesn't need to be on every host.